odoo/service/db.py

```diff
@@ -361,16 +361,11 @@ def list_dbs(force=False):
     db = odoo.sql_db.db_connect('postgres')
     with closing(db.cursor()) as cr:
         try:
-            cr.execute('SELECT current_user')
-            db_user = cr.fetchone()
-            if db_user:
-                cr.execute("select datname from pg_database where datdba=(select usesysid from pg_user where usename=%s) and not datistemplate and datallowconn and datname not in %s order by datname", (db_user, templates_list))
-            else:
-                cr.execute("select datname from pg_database where not datistemplate and datallowconn and datname not in %s order by datname", (templates_list,))
+            cr.execute("select datname from pg_database where datdba=(select usesysid from pg_user where usename=current_user) and not datistemplate and datallowconn and datname not in %s order by datname", (templates_list,))
```

Comment:
For the record, I think this will not include databases owned by a group (role) the current user belongs to. This would require a JOIN with pg_roles and pg_auth_members.

But let's keep it simple, this query is not intended to be used in production, where we recommended that:
 - the list of databases be hard-coded and passed using startup parameter `-d`
 - the owner of the databases should **not** be the one Odoo connects with

This simple query should be enough for most dev/test environments.

Reply:
Do you mean something like that :
```sql
select datname from pg_database where (datdba=(select usesysid from pg_user where usename=current_user) or datdba=(select pg_auth_members.roleid from pg_auth_members where member=(select usesysid from pg_user where usename=current_user))) and not datistemplate and datallowconn;
```
I didn't use pg_roles, so I'm not sure that is what you meant.

Reply:
Yes something like that, possibly with support for indirect role membership ;-) But don't bother, it would make the query even uglier and really looks like over-engineering.